### ISSUE TITLE
Is it possible to disable Milo rules?

### ISSUE THREAD

**Issue by nicoleescobar** [wawandco/milo#77]
Hello!, I'm working on a Hugo website with Tailwind CSS integrated, and there are a few components that have SVGs set on the HTML, the SVGs tags have an attribute called "viewBox" and it is firing the attr-lowercase rule and that's why I've been bitten by Milo.

**Comment by paganotoni** [comment]
Ah! This is one that I've seen as well. Yes, you can disable rules by using the `.milo.yml` file but I'm thinking if we should consider SVG and how do we deal with it. 

cc @larrymjordan  @tatang26

**Comment by larrymjordan** [comment]
@paganotoni  agreed!, I think we can add  all of already-known SVG attributes to a whitelist and apply it only to SVG elements, probably that would solve this issue, however your statement makes me wondering what other cases we should cover  and give SVG elements a different treatment. Any thoughts?

**Comment by paganotoni** [comment]
That sounds good to me. Probably as a side topic I wonder if the HTML standard or the Go HTML parser we have considers SVG document sections any different than the rest of the document and that something we could use to know the element we're seeing is inside SVG.
